odml/tools/odmlparser.py

```diff
@@ -88,6 +89,10 @@ def to_string(self, odml_document):
         return string_doc
 
 
+# Required to serialize datetime.time as string objects
+def YAMLTimeSerializer(dumper, data):
+        return dumper.represent_scalar('tag:yaml.org,2002:str', str(data))
```

Comment:
```suggestion
    return dumper.represent_scalar('tag:yaml.org,2002:str', str(data))
```

Reply:
Ach... I did this using vim, and not my usual IDE. I'll fix this as well. Is it fine if I squash and merge the future commits (to fix the errors and this issue), just to make the commit history clean?

Reply:
Sure.  No problem.

(Sidenote: You can get linting in vim with the jedi or ALE plugins :) )

Reply:
Sure... I'll give them a try. Thanks!